docker/README.md

```diff
@@ -103,30 +114,86 @@ You may also want to modify the `-p 80:80` to utilize other ports instead. Refer
 **Note**: if you're using the GPU Docker, skip ahead to the next section.
 
 ```bash
-docker run -it -p 80:80 mms_image:latest
+$ docker run -i -t -d -p 80:80 --name mms -v /home/user/models:/models mms_image:latest
+# This runs the docker image in a detached mode
 ```
-Now that you have a prompt inside the container, the final step is to run the following command:
 
+To run the MMS with $HOSTNAME as its endpoint, or for `nginx's server_name` to be configured to `$HOSTNAME` run the following command.
 ```bash
-cd mms_docker_cpu && ./launch.sh
+$ echo $HOSTNAME
+user.ant.amazon.com
+
+$ docker run -i -t -d -p 80:80 --name mms -v /home/user/models:/models -e MMS_HOST=$HOSTNAME mms_image:latest
+```
+The above command lets you run inference with `$HOSTNAME` as 'server_name'.  
```

Comment:
1. put the explanatory comments before the commands
2. localhost or loopback addr should work just fine. why tell user to lookup hostname and then use it?

Reply:
When nginx was configured with server_name as "localhost" I wasn't able to reach it with curl -X POST http://127.0.0.1:80/squeezenet/predict -F "data=@kitten.jpg". I get 404 error. So, I thought customers would want to use their public-ip to run inference.